src/data/gitmojis.json

```
@@ -432,6 +432,13 @@
 		  "code": ":label:",
 		  "description": "Adding or updating types (Flow, TypeScript)",
 		  "name": "label"		
+		},
+		{
+			"emoji": "🆎",
+			"entity": "U+1F18E",
+			"code": ":ab_test:",
```

isn't the code `:ab:`?
```suggestion
			"code": ":ab:",
```

---

🆎 
:ab_test:

---

It could be anything really, added "_test" in to make it clearer, else ":ab" might be confused with spelling or something else. But welcome for thoughts from others.

---

that's the catch. I don't think it can be anything, all the labels on here match with things that github understands as emojis, and so, substitute the text with the Unicode char.
And for the emoji that you choose, the "code" or "label", is actually `:ab:` which github translates as :ab: whereas `:ab_test:` translates as the literal :ab_test:

but I could be wrong here, just stating this with the intention to help :grinning:

---

No no you're absolutely right to question the name 👍 !

I think that explanation would satisfy a reason to make it :ab: since it matches with Github.

I'll update it accordingly.

---

happy to help 😊